### ISSUE TITLE
[jshow] Body copy text does not meet wcag 2.0 aa standards

### ISSUE THREAD

**Issue by webcoast** [joomla/joomla-websites#136]
Description in tab content in body copy eg https://joomla-showcase.octopoos-dev.com/browse-sites/www-sunliner-com-au.html isn't strong enough. Color #7abdbe against #fffffff needs to be darker.

![screenshot-joomla-showcase octopoos-dev com 2015-10-10 16-36-53](https://cloud.githubusercontent.com/assets/1870107/10409675/3cf0acda-6f6d-11e5-84ee-223bd07e7dae.jpg)

**Comment by MAT978** [comment]
Hello,
The Joomla! Showcase Directory is still under development.
thanks

**Comment by pbwebdev** [comment]
This is from #joomladayau. Teaching group of people how to contribute to Joomla by reporting issues and test for accessibility.

Showcase team will pick them up from here and make the changes on the new dev site.

Thanks @MAT978

**Comment by sebastienlapoux** [comment]
Fixed. Thanks.
